platform/ui/src/components/AdvancedToolbox/AdvancedToolbox.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import classnames from 'classnames';
+import { PanelSection, Icon, Tooltip } from '../../components';
+import ToolSettings from './ToolSettings';
+
+const AdvancedToolbox = ({ title, items }) => {
+  const [isActive, setIsActive] = useState(null);
```

Comment:
`isActive` sounds like a boolean to me, yet I believe this state is for the name of the active item. So consider using `[activeItemName, setActiveItemName]` instead.